.t9n.yml

```
@@ -17,5 +18,5 @@ exclude_files:
   - node_modules
   - vendor
   - tools
-  - mails
+  - mails/[a-zA-Z][a-zA-Z]/
```

```suggestion
  - mails/[a-zA-Z]{2}/
```
If there is only two characters

---

I tried this at first (of course) but it doesn't work because these regex are then used within a `$finder->notPath('{^'.$item.'}');`
https://github.com/PrestaShop/TranslationToolsBundle/blob/90e17a65667ba94ae3d9cd6c1fda1348d8669474/Translation/Extractor/ChainExtractor.php#L72

---

I tried it again just in case, and it works! (I didn't have the las `/` at the time which is why it didn't work)